cleverpush/src/main/java/com/cleverpush/GeofenceBroadcastReceiver.java

```diff
@@ -0,0 +1,115 @@
+package com.cleverpush;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.CountDownTimer;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import com.cleverpush.util.Logger;
+import com.google.android.gms.location.Geofence;
+import com.google.android.gms.location.GeofenceStatusCodes;
+import com.google.android.gms.location.GeofencingEvent;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GeofenceBroadcastReceiver extends BroadcastReceiver {
+    private double delay;
+    private final ArrayList<Double> mDelayList = new ArrayList();
+    private int transition;
+    private String channelId;
+    private String subscriptionId;
+    private String transitionState;
+    int position;
+    boolean mCompleted = false;
+    CountDownTimer cTimer;
+
+    public void onReceive(Context context, Intent intent) {
+        GeofencingEvent geofencingEvent = GeofencingEvent.fromIntent(intent);
+        if (geofencingEvent.hasError()) {
+            String errorMessage = GeofenceStatusCodes.getStatusCodeString(geofencingEvent.getErrorCode());
+            Log.e("CPErrorMessage", errorMessage);
```

Comment:
Use `Logger` class and no "CPErrorMessage needed

Reply:
okay removed Log